src/nyc_trees/apps/survey/views.py

```diff
@@ -172,3 +172,7 @@ def admin_extend_blockface_reservation(request, blockface_id):
 def admin_blockface_available(request, blockface_id):
     # TODO: implement
     pass
+
+
+def reservations_instructions(request):
+    pass
```

Comment:
There is no need for this "empty" view.  The `render_template` decorator can work without actually decorating anything.